torch/utils/data/dataloader.py

```diff
@@ -275,7 +295,11 @@ def __setattr__(self, attr, val):
 
         super(DataLoader, self).__setattr__(attr, val)
 
-    def __iter__(self):
+    # We quote '_BaseDataLoaderIter' since it isn't defined yet and the definition can't be moved up
+    # since '_BaseDataLoaderIter' references 'DataLoader'. In mypy 0.720 and newer a new semantic
+    # analyzer is used that obviates the need for this but we leave the quoting in to support older
+    # versions of mypy
```

Comment:
This is no longer necessary - mypy 0.770 is the minimum requirement.

Reply:
If I remove the quotes, I'm still getting `NameError: name '_BaseDataLoaderIter' is not defined` from mypy 0.770. I have removed the part of the comment about newer versions of mypy handling this

Reply:
Sounds good, thanks